app/controllers/application_controller/explorer.rb

```diff
@@ -10,12 +10,14 @@ def build_replaced_trees(replace_trees, valid_values)
   end
 
   def try_build_tree(tree_symbol)
-    method_name = "build_#{tree_symbol}_tree"
-    return unless respond_to?(method_name, true)
-    build_method = method(method_name)
-    # FIXME: This is temporary, we actually need to remove all the build_*_tree methods and
-    # use the Feature::build_tree instead.
-    build_method.arity == 1 ? build_method.call(tree_symbol) : build_method.call
+    # Legacy support for build_*_tree methods
+    # FIXME: delete this after all of them were converted (remaining: build_reports_tree)
+    method_name = :"build_#{tree_symbol}_tree"
```

Comment:
is `build_reports_tree` the only one left?

is there a reason to build `method_name` dynamically?

Reply:
Yup, this was the old way, wanted to keep it as a memento instead of explicit calls. This way we might have bigger motivation to get rid of it.

Reply:
Should I make it *"static"* @kbrock?

Reply:
@skateman lol. So you want to keep the code ugly so we will be encouraged to get rid of this whole method?

seems like a turkey response, but good enough for me.

Reply:
@kbrock well, yes :) and I also wanted to do minimal changes here...